cmd/main.go

```diff
@@ -67,6 +69,11 @@ func main() {
 		if len(subset) == 0 {
 			log.Fatalf("Unable to find an element with the class '%s'\n", *byClass)
 		}
+	} else if *byTagName != "" {
+		subset = restify.FindSubnetByTagName(root, atom.Lookup([]byte(*byTagName)))
```

Comment:
Please follow the pattern of the others where second arg should only be `*byTagName`. The atom details should be encapsulated inside the function.